bedrock/releasenotes/views.py

```diff
@@ -173,15 +176,17 @@ def releases_index(request, product):
         # The version numbering scheme of Firefox changes sometimes. The second
         # number has not been used since Firefox 4, then reintroduced with
         # Firefox ESR 24 (Bug 870540). On this index page, 24.1.x should be
-        # fallen under 24.0. This patter is a tricky part.
-        major_pattern = r'^' + \
-            re.escape(
-                ('%s' if major_version < 4 else '%g') % round(major_version, 1))
+        # fallen under 24.0. This pattern is a tricky part.
+        if int(major_version) in esr_major_versions:
```

Comment:
A bit of repitition here. the previous version was a little crazy, but what about something like:

```python
converter = '%s' if int(major_version) in esr_major_versions else '%g'
major_pattern = r'^' + re.escape(converter % round(major_version, 1))
```